xarray/core/dataset.py

```diff
@@ -1742,6 +1742,33 @@ def set_coords(self: T_Dataset, names: Hashable | Iterable[Hashable]) -> T_Datas
         names : hashable or iterable of hashable
             Name(s) of variables in this dataset to convert into coordinates.
 
+        Examples
+        --------
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "temperature": ("time", [25, 30, 27]),
```

Comment:
Nice simple example!

I suggest changing "temperature" to "pressure", and changing the values to be plausibly measured in bar or kPa, because pressure coordinates are a real thing for atmospheric scientists that are commonly used.